app/src/main/res/xml/main_settings.xml

```diff
@@ -1,41 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PreferenceScreen
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:key="general_preferences"
     android:title="@string/settings">
 
     <PreferenceScreen
+        app:iconSpaceReserved="false"
```

Comment:
Why did you add this here?
In case we want to add this lines to every settings fragment, we there might be a better approach: https://stackoverflow.com/questions/51518758/preferencefragmentcompat-has-padding-on-preferencecategory-that-i-cant-get-rid#answer-52960668